plansys2_executor/src/plansys2_executor/ExecutorNode.cpp

```diff
@@ -274,10 +274,12 @@ ExecutorNode::execute(const std::shared_ptr<GoalHandleExecutePlan> goal_handle)
   if (status == BT::NodeStatus::FAILURE) {
     tree.haltTree();
     RCLCPP_ERROR(get_logger(), "Executor BT finished with FAILURE state");
+    result->success = false;
+  } else {
+    result->success = true;
   }
 
   result->action_execution_status = get_feedback_info(action_map);
-  result->success = true;
```

Comment:
maybe we could write?

```
result->success = status == BT::NodeStatus::SUCCESS;
```

Reply:
Of course.  b522c0b

Reply:
Of course.   56ef8b6729fb7f3d9ae551c0d23cb4e17b522c0b